openfisca_us/variables/gov/ssa/ssi/eligibility/income/deemed/from_ineligible_parent/ssi_unearned_income_deemed_from_ineligible_parent.py

```diff
@@ -0,0 +1,50 @@
+from openfisca_us.model_api import *
+from openfisca_us.variables.gov.ssa.ssi.eligibility.income._apply_ssi_exclusions import (
+    _apply_ssi_exclusions,
+)
+
+
+class ssi_unearned_income_deemed_from_ineligible_parent(Variable):
+    value_type = float
+    entity = Person
+    label = "SSI unearned income (deemed from ineligible parent)"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://www.law.cornell.edu/cfr/text/20/416.1165"
+
+    def formula(person, period, parameters):
+        eligible_child = person("is_ssi_aged_blind_disabled", period) & person(
+            "is_child", period
+        )
+        ineligible_parent = person("is_ssi_ineligible_parent", period)
+
```

Comment:
To use below in a few places:
```suggestion
    tax_unit = person.tax_unit
```
Also could you share where the law defines the unit as a tax filing unit? For example, do elderly dependents count? Reads elsewhere as family.

Reply:
Hmm, I read as tax unit but struggling to find a definition now. I think it's probably worth merging with tax unit, and adding the extra logic to determine who's the parent of which children if it is indeed family (filed #844).